### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewSearch_ScreenReader: Screenreader does not read the necessary headings on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52276]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Search"(Cntrl+Shift+F) button.
3. Navigate to complementary landmark -"Search" section and read the content "Search" using basic navigation.

**Actual:**
Screenreader does not read the content "Search" as appopriate heading which is so.

**Same issue has been observed with JAWS.**

**Expected:**
Screenreader should read- "Search" as heading with appropriate heading level.

**Recommendations:**
Add appropriate native heading level tag to the content "Search".
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the appropriate headings present on the screen.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_ViewSearch_ScreenReader_Headings.pptx](https://github.com/Microsoft/vscode/files/2114265/A11y_VSCode_ViewSearch_ScreenReader_Headings.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
> Navigate to complementary landmark -"Search" section and read the content "Search" 

Sorry, I don't know what this means. When I press ctrl+shift+f, it reads "Search". What is the complementary landmark?

**Comment by AccessibilityTestingTeam-TCS** [comment]
Basically the point here is that the headings like "Search", "Explorer" etc. are not identified as headings. They should have proper heading levels. The side bar content - "Search", "Explorer" etc. can be accessed using landmarks (**D** command in NVDA for navigating to landmarks) and then try to read headings inside that using (**H** command in NVDA for headings). There are no heading levels found for above mentioned headings. 


![headings](https://user-images.githubusercontent.com/39204909/41893141-60415db0-7938-11e8-94e0-552b03f79d2a.png)

Hope this explains the needed.

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51760
